libraries/apollo-compiler/src/test/graphql/com/example/measurements

```diff
@@ -2,244 +2,11 @@
 // If you updated the codegen and test fixtures, you should commit this file too.
 
 Test:                                                                                      Total LOC:
-aggregate-all                                                                                  193357
-aggregate-kotlin-responseBased                                                                  61239
-aggregate-kotlin-operationBased                                                                 40535
+aggregate-all                                                                                    1326
+aggregate-kotlin-responseBased                                                                    628
+aggregate-kotlin-operationBased                                                                     0
 aggregate-kotlin-compat                                                                             0
-aggregate-java-operationBased                                                                   91583
+aggregate-java-operationBased                                                                     698
```

Comment:
This is probably the value for a single test. Do you mind regenerating for all tests?